packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -81,8 +81,17 @@ def get_measurement_service_stub(
             )
         else:
             raise
-    channel = channel_pool.get_channel(service_location.insecure_address)
-    return v2_measurement_service_pb2_grpc.MeasurementServiceStub(channel)
+    channel = channel_pool.get_channel(resolve_service_response.service_location.insecure_address)
+    version = (
+        resolve_service_response.service_info.versions[0]
```

Comment:
FYI: I assume this call only returns one version, not all available versions.

Reply:
yes, only one version will be returned.

Reply:
Either way, the first version returned should be the newest version which generally seems like what the client generator should use given the UI doesn't present the user a choice.